config/rbac/role.yaml

```diff
@@ -38,3 +34,9 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - ocs.openshift.io
+  resources:
+  - storageclusters
+  verbs:
+  - '*'
```

Comment:
This will conflict with the recent add-on PR for RBAC changes. 
If we can wait for it to be merged this changed will be obsolete.

Reply:
That will be taken care of with a rebase